### PR TITLE
Update features for document text fields

### PR DESCRIPTION
**Pull request by qduanmu**

### CODE REVIEW

File: features/pages/home_page.py [hunk 1]
```diff
@@ -59,8 +59,7 @@ def click_flaw_check_all_checkbox(self):
     def check_is_all_flaw_selected(self):
         flaw_rows = find_elements_in_page_factory(self, "flawRow")
         flaw_checkboxes = find_elements_in_page_factory(self, "flawCheckBox")
-        assert len(flaw_rows) == len(flaw_checkboxes), 'Incorrect checkbox count'
-
+        assert len(flaw_rows) + 1 == len(flaw_checkboxes), 'Incorrect checkbox count'
```
Comment (RichardXuan) [inline features/pages/home_page.py:62]: Why +1

File: features/pages/flaw_detail_page.py [hunk 2]
```diff
@@ -8,10 +9,17 @@ def __init__(self, driver):
...
+        "summaryTextWindow": ("XPATH", "(//textarea[@class='form-control col-9 d-inline-block'])[1]"),
+        "descriptionText": ("XPATH", "//span[text()='Description']"),
+        "descriptionTextWindow": ("XPATH", "(//textarea[@class='form-control col-9 d-inline-block'])[2]"),
         "statementText": ("XPATH", "//span[text()='Statement']"),
         "statementTextWindow": ("XPATH", "(//textarea[@class='form-control col-9 d-inline-block'])[3]"),
+        "mitigationText": ("XPATH", "//span[text()='Mitigation']"),
+        "mitigationTextWindow": ("XPATH", "(//textarea[@class='form-control col-9 d-inline-block'])[4]"),
+        "addCommentBtn": ("XPATH", "//button[contains(text(), 'Comment')]"),
+        "commentTextWindow": ("XPATH", "(//textarea[@class='form-control col-9 d-inline-block'])[5]"),
+        "saveBtn": ("XPATH", '//button[text()="Save Changes"]'),
```
Comment (huiwangredhat) [inline features/pages/flaw_detail_page.py:22]: It is duplicated with line 23.

Comment (huiwangredhat) [reply features/pages/flaw_detail_page.py:22]: LGTM except for the last comment.

**Review by RichardXuan — APPROVED** [review]
LGTM

**Pull request merged**